ngraph/test/CMakeLists.txt

```diff
@@ -488,7 +487,11 @@ file(GLOB FRONTEND_TESTS_SRC ${CMAKE_CURRENT_SOURCE_DIR}/frontend/frontend_manag
 set(SRC ${FRONTEND_TESTS_SRC} ${SRC})
 
 file(GLOB FRONTEND_SHARED_TESTS_SRC ${CMAKE_CURRENT_SOURCE_DIR}/frontend/shared/src/*.cpp)
+file(GLOB FRONTEND_SHARED_TESTS_HDR ${CMAKE_CURRENT_SOURCE_DIR}/frontend/shared/include/*.hpp)
 set(SRC ${FRONTEND_SHARED_TESTS_SRC} ${SRC})
+set(HDR ${FRONTEND_SHARED_TESTS_HDR} ${HDR})
```

Comment:
As I see `${HDR}` is empty. Can you use `FRONTEND_SHARED_TESTS_HDR` or add headers to unit-test target and use `FOR_TARGET` option?

Reply:
Removed HDR. I thought I'll use 'HDR' for future 'paddlepaddle-frontend' tests, but realized that pdpd-specific tests don't have any headers.
I also moved 'Frontend-related' variables upper in the file, and kept 'clang' target definition on old place. This is needed because later 'SRC' will contain auto-generated ONNX code which don't follow to 'clang' format